envs/picking_env/picking_task.py

```diff
@@ -120,9 +138,34 @@ def _get_obs(self):
 
     def _get_reward(self, state):
         object_pos = state[4:]
-        dist_to_goal = np.linalg.norm(object_pos - self._goal_pos)
-        success = dist_to_goal < self._dist_threshold
-        reward = None
+        obj_dist_to_goal = np.linalg.norm(object_pos - self._goal_pos)
+        success = obj_dist_to_goal < self._dist_threshold
+        if self.reward_type == "sparse_handcrafted":
+            reward = success
+        elif self.reward_type == "dense_handcrafted":
+            if not self._subgoal0_reached:
```

Comment:
isnt subgoal0 starting location. Would it not be true at reset. Do we need this check then?

Reply:
lol - good point; i think i meant subgoal 1

Reply:
lemme fix